AmplifyTools/AmplifyXcode/Sources/AmplifyXcode/CLICommandReportable.swift

```diff
@@ -25,16 +25,18 @@ extension CLICommandReportable {
     }
 
     private func reportCommandFailure(_ error: AmplifyCommandError) {
-        print(error.debugDescription)
+        print("🚫 \(error.debugDescription)")
     }
 
     func report(result: AmplifyCommandResult) {
         switch result {
         case .success(let intermediateRes):
             reportCommandSuccess(for: intermediateRes)
+            exit(EXIT_SUCCESS)
```

Comment:
Let's not put such a drastic side effect (exiting the program) inside a `report` method. Call `exit` explicitly at the call site where you're reporting, or less preferably, refactor into a `reportAndExit` method that does both.

Reply:
Sure, I'll move it to the call site. I've also just found that `ArgumentParser` library exposes an `Exit.failure` error that can be thrown from the `run` method.